internal/commands/common.go

```diff
@@ -22,6 +22,7 @@ var (
 	infoCommandName   = "info"
 	listCommandName   = "list"
 	createCommandName = "create"
+	dockerCommandName = "create-docker"
```

Comment:
Can we update this command to `create-with-registry` (in order to be closer to the API endpoint naming `CreateBuildRegistry`)